.flake8

```diff
@@ -1,5 +1,5 @@
 [flake8]
-ignore = F, E203, W605, E501, W503, D100, D104
+ignore = F, E203, W605, E501, W503, D100, D104, C408
```

Comment:
Why do you decide to ignore C408?

Reply:
Because in cases like this
https://github.com/tinkoff-ai/etna/blob/be1b719dd3d32ebb8d966c22eeac8309537e4792/etna/models/nn/deepar.py#L133-L139

It would be easier to read dict(key=value) approach, than {"key": value} in my opinion

Reply:
What do you think?